app/Http/Controllers/Wiki/CountriesController.php

```diff
@@ -128,10 +128,10 @@ public function search($search_text = '')
             return $this->setStatusCode(400)->replyWithError(trans('api.search_errors_400'));
         }
 
-        $cache_params = $this->removeSpaceFromCacheParameters(
-            [$GLOBALS['i18n_iso'], $limit, $page, $formatted_search_cache]
-        );
-        $countries = cacheRemember('countries', $cache_params, now()->addDay(), function () use ($limit, $page, $formatted_search) {
+        $cache_params = [$GLOBALS['i18n_iso'], $limit, $page, $formatted_search_cache];
+        $cache_key = generateCacheSafeKey('countries', $cache_params);
```

Comment:
We will create a safe key before we call the cacheRememberByKey method. For now, it will be implemented for the searching endpoints